nixos/modules/system/boot/luksroot.nix

```diff
@@ -158,6 +158,21 @@ let
       wait_target "header" ${dev.header} || die "${dev.header} is unavailable"
     ''}
 
+    try_empty_passphrase() {
+        ${if dev.tryEmptyPassphrase then ''
+             echo "Trying empty passphrase!"
+             echo "" | ${csopen}
+             cs_status=$?
+             if [ $cs_status -eq 0 ]; then
+                 echo "Empty passphrase succeeded"
```

Comment:
No need for the verbosity with "Empty passphrase succeeded". Plus you can just leave `${csopen}` as the last command, and that will cause its exit code to be returned.